app/controllers/concerns/claim_session_timeout.rb

```diff
@@ -1,25 +1,6 @@
-module ClaimSessionTimeout
-  CLAIM_TIMEOUT_LENGTH_IN_MINUTES = 30
-
+module ClaimSessionTimeout # TODO: rename this class - SessionManager?
   def clear_claim_session
-    session.delete(:claim_postcode)
-    session.delete(:claim_address_line_1)
```

Comment:
these were only ever used by the `app/views/claims/no_address_found.html.erb` template, which was never called